### PR TITLE
python-tabulate 0.9.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by cho-m** [comment]
Looks like upstream got rid of old `setup.py` installation in favor of `pyproject.toml`.

Sadly, as more Python packages migrate to this approach, it makes things trickier for us as our Python formulae packaging approach doesn't match officially supported way. We can't run `python3 setup.py ...` anymore. This may work with `pip install` or `python -m build` (using extra dependency).

The latter can be done using [`python-build`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/python-build.rb). ArchLinux has an example of using `--no-isolation`: https://github.com/archlinux/svntogit-community/blob/packages/python-tabulate/trunk/PKGBUILD#L25

I don't know if there is a clean way of using `pip install` for non-virtualenv installations. May need to figure out arguments to imitate existing installations.

**Comment by carlocab** [comment]
We should also get rid of `libpython-tabulate`.

We added it as a way to work around the versioned dependency audit, but that should no longer be an issue.

File: Formula/libpython-tabulate.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class LibpythonTabulate < Formula
```
Comment (carlocab) [inline Formula/libpython-tabulate.rb:1]: https://github.com/Homebrew/homebrew-core/pull/112518#issuecomment-1272512678

**Review by carlocab — DISMISSED** [review]

File: Formula/libpython-tabulate.rb (hunk 1 above)
Comment (branchvincent) [reply Formula/libpython-tabulate.rb]: Updated, still need to figure out which runtime python to use that doesn't cause audit failures. The following fails with `References to python@3.9 should match the specified python dependency`:

```ruby
depends_on "python@3.9" => [:build, :test]
depends_on "python@3.10"
```

With `uses_from_macos`, I'm wondering how to refer to the runtime python in the install block (I don't think we can simply use PATH's python)

```ruby
depends_on "python@3.9" => [:build, :test]
depends_on "python@3.10" => [:build, :test]

uses_from_macos "python"
```

Any ideas?

**Comment by Porkepix** [comment]
@branchvincent Should also be removed from here: https://github.com/Homebrew/homebrew-core/blob/master/synced_versions_formulae.json#L53

**Review by SMillerDev — DISMISSED** [review]

**Comment by branchvincent** [comment]
Needs a rebase for #116473

**Comment by chenrui333** [comment]
make a 🏃

**Comment by chenrui333** [comment]
changes lgtm.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**